xcat-inventory/xcclient/allien/resources/security.py

```diff
@@ -161,6 +164,28 @@ def delete(self, id):
 
         return None, 200
 
+    @ns.expect(sec_post_resource)
+    def post(self, id):
+        """modify a passwd object"""
```

Comment:
The comments will be shown on OpenAPI doc, keep the same , use `user secret`, instead of `passwd`